requirements/docs.txt

```diff
@@ -12,3 +12,4 @@ sphinx-autodoc-typehints
 sphinx-paramlinks<0.4.0
 sphinx-togglebutton
 sphinx-copybutton
+fsspec[http]
```

Comment:
this isn't the right place, here go only dependencies for generating docs pages.
it's not needed, you can drop it :)

Reply:
Thanks @awaelchli for review!  
Without this `docs.txt`, [build-Docs](https://app.circleci.com/pipelines/github/PyTorchLightning/pytorch-lightning/23042/workflows/9474789a-1827-4380-a349-6158da9231e9/jobs/84808) CI failed.  
Do you know how to fix it without `docs.txt` modification?

Reply:
I suspect that the package doesn't get mocked because you added the brackets ["http"] to the file. This function here
https://github.com/PyTorchLightning/pytorch-lightning/blob/d20fd8e5ab1a52747fee2cd53290a679d8b726d0/docs/source/conf.py#L296

probably can't parse it.
cc @Borda

Reply:
one quick hack you can try is to add 

```python
PACKAGE_MAPPING = {
    "fsspec["http"]": "fsspec"
```

in the package mapping below (line 308)

Reply:
Fixed with `package_list_from_file` improvement.  
@awaelchli thank you very very much, this error stack me more than one month, I am very relieved...

Reply:
@tarepan oh oh, sorry for that, I wish I had seen it earlier :( 
great work btw!